dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/ConfigServerXMLOverrideTests20.java

```diff
@@ -175,7 +197,12 @@ public String getLogMethodName() {
 
     @After
     public void stopOverrideServer() throws Exception {
-        super.stopServer();
+        String methodName = testName.getMethodName();
+        if (methodName != null && methodName.contains(DEFAULT_HOST_WITH_APP_PORT)) {
+            super.stopServer(true, "CWWKT0015W");
+        } else {
+            super.stopServer();
```

Comment:
Why not call `stopServer(true);` here?

Reply:
super.stopServer() will call super.stopServer(true)